lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -19,7 +18,7 @@
     </form>
 
     <a href="" class="action-primary action-create">
-        <i class="icon fa fa-plus"></i>
+        <i class="icon fa fa-plus" aria-hidden="true"></i>
```

Comment:
Since icons are visual and since there's text on this button that describes the action, we want to aurally hide this icon.